crates/rattler-bin/src/commands/create.rs

```diff
@@ -31,38 +27,38 @@ pub async fn create(opt: Opt) -> anyhow::Result<()> {
         .map_err(|e| anyhow::anyhow!("could not create cache directory: {}", e))?;
 
     // Get the channels to download
-    let channels = opt
+    let _channels = opt
         .channels
         .unwrap_or_else(|| vec![String::from("conda-forge")])
         .into_iter()
         .map(|channel_str| Channel::from_str(&channel_str, &channel_config))
         .collect::<Result<Vec<_>, _>>()?;
 
-    // Download all repo data from the channels and create an index
-    let repo_data_per_source = MultiRequestRepoDataBuilder::default()
-        .set_cache_dir(&cache_dir)
-        .set_listener(terminal_progress())
-        .set_fail_fast(false)
-        .add_channels(channels)
-        .request()
-        .await;
-
-    // Error out if fetching one of the sources resulted in an error.
-    let repo_data = repo_data_per_source
-        .into_iter()
-        .map(|(channel, _, result)| result.map(|data| (channel, data)))
-        .collect::<Result<Vec<_>, _>>()?;
-
-    let solver_problem = SolverProblem {
-        channels: repo_data
-            .iter()
-            .map(|(channel, repodata)| (channel.base_url().to_string(), repodata))
-            .collect(),
-        specs,
-    };
-
-    let result = solver_problem.solve()?;
-    println!("{:#?}", result);
+    // // Download all repo data from the channels and create an index
+    // let repo_data_per_source = MultiRequestRepoDataBuilder::default()
+    //     .set_cache_dir(&cache_dir)
+    //     .set_listener(terminal_progress())
+    //     .set_fail_fast(false)
+    //     .add_channels(channels)
+    //     .request()
+    //     .await;
+    //
+    // // Error out if fetching one of the sources resulted in an error.
+    // let repo_data = repo_data_per_source
+    //     .into_iter()
+    //     .map(|(channel, _, result)| result.map(|data| (channel, data)))
+    //     .collect::<Result<Vec<_>, _>>()?;
+    //
+    // let solver_problem = SolverProblem {
+    //     channels: repo_data
+    //         .iter()
+    //         .map(|(channel, repodata)| (channel.base_url().to_string(), repodata))
+    //         .collect(),
+    //     specs,
+    // };
+    //
+    // let result = solver_problem.solve()?;
+    // println!("{:#?}", result);
```

Comment:
Don't forget to fix the create command to use the new feature

Reply:
Im planning on refactoring that completely once we have a more stable solver interface. I didn't think this was used (since it doesn't actually do much), so I simply broke it. 

Are you use this?

Reply:
I am not using it, but I tried to ensure it keeps working after my solver interface changes. I was thinking if you keep it working here it will be easier to resolve the merge conflicts later in a way that makes sense

Reply:
(if you think it is easier to just ignore the `create` command for the time being, we can leave it broken and fix it later as well)